src/sema/tags.rs

```diff
@@ -11,7 +11,7 @@ pub fn resolve_tags(
     tags: &[DocComment],
     params: Option<&[Parameter]>,
     returns: Option<&[Parameter]>,
-    bases: Option<&[String]>,
+    bases: Option<Vec<usize>>,
```

Comment:
Nit: This type argument can probably be flattened into just a `Vec` (I can't see a case where passing `Some` empty Vec makes sense here, and in fact the code does nowhere handle this case explicitly, making it superfluous)

Reply:
`@inheritdoc` only makes sense for some items. For others, e.g. structs outside defined out of contracts, you want the diagnostic "@inheritdoc not allowed".